arrow-udf-flight/src/lib.rs

```diff
@@ -68,6 +68,11 @@ impl Client {
         })
     }
 
+    pub fn max_decoding_message_size(mut self, limit: usize) -> Self {
+        self.client = self.client.max_decoding_message_size(limit);
```

Comment:
I'm considering whether we can allow users to use their own client instead. What do you think?

Reply:
Yes, that was my initial thought as well actually. It's be more elegant.

Reply:
Great, let's proceed this way. I think this can prevent us from exposing many unnecessary public APIs.